tests/modules/sightings/resources/test_create_sighting.py

```diff
@@ -483,10 +483,11 @@ def test_create_old_sighting(flask_app_client, researcher_1):
     )
 
 
+# with edm gone this is kind of just redundant/bonus testing
```

Comment:
Yeah I was wondering quite howuseful this mixed testing was anymore too.

Reply:
if my hacky grepping is any indication, the only novel thing in this test is a `PATCH` that included `timeSpecificity`.  does that warrant keeping it around?  i dunno.  :sweat_smile:  would i _miss it_?  likely not.